apps/emqx/include/emqx.hrl

```diff
@@ -17,25 +17,8 @@
 -ifndef(EMQ_X_HRL).
 -define(EMQ_X_HRL, true).
 
-%%--------------------------------------------------------------------
-%% Common
-%%--------------------------------------------------------------------
-
--define(Otherwise, true).
-
--define(COMMON_SHARD, emqx_common_shard).
--define(SHARED_SUB_SHARD, emqx_shared_sub_shard).
--define(MOD_DELAYED_SHARD, emqx_delayed_shard).
--define(CM_SHARD, emqx_cm_shard).
+-include_lib("emqx_machine/include/emqx_machine.hrl").
```

Comment:
emqx should not know the existence of emqx_machine.

emqx app is intended to be used as a dependency for other rebar3 projects.

Reply:
good point, I should defined the in emqx, let emqx_machine call emqx's funcion.